win-odbc64/odbcclient/drvr35/cstmt.cpp

```diff
@@ -3389,7 +3389,7 @@ BOOL CStmt::setFetchOutputPerf(long rowsFetched, SQL_DataValue_def*& outputDataV
     m_FetchDataValue.numberOfRows = rowsFetched;
     if(rowsFetched > 0)
     {
-        m_FetchDataValue.rowAddress = new unsigned long[rowsFetched];
+        m_FetchDataValue.rowAddress = new LONG_PTR[rowsFetched];

```

Comment:
The issue here is that the long is 4 bytes in Windows x64.  windows can ensure the LONG_PTR is 8 bytes in x64 system and 4 bytes in x32 system. I use LONG_PTR because of a lot Windows API use it . There is no VOID_PTR type.
I can use (void *) instead if you insist .